scripts/test_database

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env bash
+set -e
+if [ $# -lt 3 ]
+  then
+    cat <<EOF
+Please provide 3 arguments 
+	- a database type (mysql|postgres)
+	- image path (mysql:8)
+	- a path to a docker-compose tempalte
+EG:
+  ./scripts/test_database postgres postgres:11 ./resources/test/docker-compose-psql-tmpl.yaml
+EOF
+fi
+
+if ! [[ $1 == "postgres" || $1 == "mysql" ]]; then
+	echo "First argument should either be postgres or mysql"
+	exit 1;
+fi
+
+
+TARGET_COMPOSE="gen-docker-compose-$1.yml"
+IMAGE=$2 envsubst < "$3" > "$TARGET_COMPOSE"
```

Comment:
would also early assign to names variables as early as possible. The meaning of `$3` is hard to see.

Reply:
Should be better now